practice-one/pages/index.html

```diff
@@ -561,7 +561,7 @@ <h2 class="section-title">Lets Stay In Touch</h2>
     <section class="container">
```

Comment:
Can I remove this element and make a class **container** for the footer tag?

Reply:
We need the black part that spans that part of the screen. So if you do that, there will be space on both sides